.changeset/flat-bananas-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
```

Comment:
[ask] Is this PR a patch update?

```suggestion
"stylelint": patch
```

Reply:
stylelint will report more errors so it's a minor
see https://stylelint.io/about/semantic-versioning/#minor-release

Reply:
So, "false negatives" are correct?

Reply:
This PR seems to reduce false errors, for example: 🤔 

```css
fencedframe, listbox, portal, selectlist {}
```

<img width="511" alt="image" src="https://github.com/stylelint/stylelint/assets/473530/68edf0d7-cc4a-4a3a-88e6-423fe4be6bc9">

Reply:
I don't know.
I will put `patch` to be consistent with previous false-positive PRs.